### ISSUE TITLE
bulkDelete doesn't work with API user

### ISSUE THREAD

**Issue by baermathias** [strapi/strapi#11094]
### Describe the bug
When I try to use the bulkDelete endpoint: POST `/content-manager/collection-types/:model/actions/bulkDelete`, it gives me the following response:
```json
{
  "error": "contentType.notFound"
}
```
### Steps to reproduce the behavior
1. Create a new project and a collection type e.g. `book`. Also create some entries.
2. Activate the endpoint: settings -> roles -> authenticated -> content-manager -> <span>&#9989;</span> bulkDelete 
3. POST to the endpoint `/content-manager/collection-types/book/actions/bulkDelete` with or without JSON params

### Expected behavior
Delete all entries

### System
- Node.js version: v14.15.4
- NPM version: 6.14.10
- Strapi version: 3.6.8
- Database: sqlite
- Operating system: Ubuntu 20.04

**Comment by MattieBelt** [comment]
Hey @baermathias, the `bulkDelete` endpoint is from the Internal CTM api, be careful by allowing it to be requested by authenticated users.

But because it is an internal API for the CTM, it uses the UID of the content-type, e.g. `application::book.book`, as also is used in the URL in the Content Type Manager (`/admin/plugins/content-manager/collectionType/application::book.book?page=1&pageSize=10&_sort=id:ASC`).

**Comment by derrickmehaffy** [comment]
This is correct, this endpoint is not meant to be directly interfaced with from outside the admin. The v4 will add more native bulk delete options that user can use so I'm going to set this as closed.
